### ISSUE TITLE
Introduce onhover functionality to forecasting graphs

### ISSUE THREAD

**Issue by ryales** [BrcMapsTeam/europe-15-situational-awareness#10]
UNHCR did raise a point about how they only show trends/correlations rather than actual numbers, so maybe this shouldn't be a feature?

**Comment by heidieh** [comment]
So do we still want to do this?

**Comment by ryales** [comment]
Nah, the numbers are low now anyway and I guess its not best practice to notify of the exact entries
